### PR TITLE
Clean-up

### PR DESCRIPTION
**Pull request by paescuj**
This is an overall clean-up of the code-base.

### General
- Consistent usage of terms and letter casing.
- Fix typos and enhance formatting in `README.md`.
- Use single quotes whenever possible.
- Use [JSDoc](https://jsdoc.app) to describe the functions, so the descriptions are displayed in IDE.

### Tests
- Lint test files as well & fix resulting errors and warnings.
- Use [Inline Snapshots](https://jestjs.io/docs/snapshot-testing#inline-snapshots) everywhere it makes sense.
     This means you can easily update the expected content when the output changes by running Jest with `--updateSnapshot` instead of manually updating the test file.
- Disable output so the console doesn't get polluted by it and it (outputs from `@actions/core`) won't be interpreted by the CI (see for example https://github.com/MishaKav/jest-coverage-comment/actions/runs/3291557621).

### CODE REVIEW

**Review by MishaKav — APPROVED** [review]
Thanks for the amazing job! 💪

File: jest.config.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+// Turn off stdout in tests, so console doesn't get polluted by it and core output isn't interpreted in CI
+process.stdout.write = () => {
+  return false
+}
```
Comment (MishaKav) [inline jest.config.js:4]: @paescuj I like this feature, however, it breaks the main workflow that relies on `output` for jest `coverage.txt`.
I succeeded in fixing it [here](https://github.com/MishaKav/jest-coverage-comment/pull/41/files#diff-844b50bbe484710800b98cbbff0a275b5fdb0c813eac29c7cb8a2e8f9a0d5705R49)

When I work locally and run `npm run test:coverage` - it stops showing me the `text-summary` of jest.
Do you know how to remove output in GitHub action, so we can still get the `text-summary` that is also part of printing in console?

**Comment by paescuj** [comment]
> Thanks for the amazing job! 💪

You're welcome 😃 Thank you, too! 💪

File: jest.config.js (hunk 1 above)
Comment (paescuj) [reply jest.config.js:4]: I'm sorry about this, haven't tested the coverage output 🙈
I've now submitted a better way in #42.

**Pull request merged**